app/index.html

```diff
@@ -5,13 +5,289 @@
+    <link rel="stylesheet" href="css/slick.css">
     <link rel="stylesheet" href="css/main.css">
-    <title>Document</title>
+    <title>CV_Roman_Linevych</title>
 </head>
 
 <body>
-
+    <header class="header">
+        <div class="wrapper">
+            <div class="header__content flex flex_space-between">
+                <a href="/" class="header__logo">
+                    <h2 class="title">
+                        <span class="title title__tag">&#60; </span> CV <span class="title title__tag">&#62;</span>
+                    </h2>
+                </a>
+                <div class="header__menu">
+                    <nav class="nav"><span class="nav tag nav__tag">&#60;</span> <span
+                            class="nav tag nav__tag nav__slash">/ </span> Menu <span
+                            class="nav tag nav__tag">&#62;</span>
+                        <ul class="nav__list">
+                            <li class="nav__list-item"><a class="nav__list-link" href="#summary"> .summary</a></li>
+                            <li class="nav__list-item"><a class="nav__list-link" href="#skills"> .tech_skills</a></li>
+                            <li class="nav__list-item"><a class="nav__list-link" href="#languages"> .languages</a></li>
+                            <li class="nav__list-item"><a class="nav__list-link" href="#education"> .education</a></li>
+                            <li class="nav__list-item"><a class="nav__list-link" href="#works"> .my_works</a></li>
+                            <li class="nav__list-item"><a class="nav__list-link" href="#contacts"> .contacts</a></li>
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+        </div>
+    </header>
+    <section class="banner">
+        <div class="wrapper">
+            <h1 class="banner__title _anim-items _anim-no-hide">
+                Hello, I'm Roman Linevych,<span
+                    class="banner__title banner__title_think _anim-items _anim-no-hide">FRONTEND
+                    DEVELOPER</span>
+            </h1>
+            <div class="banner__scroll uppercase">
+                Scroll down
+            </div>
+            <div class="banner__arrow-wrapper">
+                <a href="#summary"><img class="banner__arrow" src="assets/icons/arrow-down.png" alt="arrow-down"></a>
+            </div>
+        </div>
+    </section>
+    <section class="summary" id="summary">
+        <div class="wrapper">
+            <div class="summary__content">
+                <h2 class="summary__title title uppercase">
+                    <span class="title title__tag">&#60;</span> Summary <span class="title title__tag">&#62;</span>
+                </h2>
+                <div class="summary__content flex flex_space-between _anim-items _anim-show">
+                    <div class="summary__text">
+                        I want to start my career a frontend developer and gain in-depth knowledgeand understanding of
```

Comment:
I want to start my career as a frontend developer*
and gain comprehensive knowledge and...